docs/sources/dashboards/create-reports/index.md

```diff
@@ -340,11 +340,19 @@ You can configure organization-wide report settings in the **Settings** under **
 
 You can customize the branding options.
 
-Report branding:
+### Attachment settings
+
+#### PDF
 
 - **Company logo:** Company logo displayed in the report PDF. It can be configured by specifying a URL, or by uploading a file. The maximum file size is 16 MB. Defaults to the Grafana logo.
```

Comment:
```suggestion
- **Company logo URL:** Company logo displayed in the report PDF. It can be configured by specifying a URL, or by uploading a file. The maximum file size is 16 MB. Defaults to the Grafana logo.
```